solana-programs/anchor/audius-data/programs/audius-data/src/utils.rs

```diff
@@ -9,6 +9,7 @@ pub fn validate_user_authority<'info>(
     user: &Account<'info, User>,
     user_delegate_authority: &AccountInfo<'info>,
     authority: &Signer,
+    app_delegation_status: &AccountInfo<'info>,
```

Comment:
Re: naming, we should pick one of `app_delegation` or `authority_delegation` and adhere to it. My preference for the latter as it is clearer.

Reply:
yeah forgot to change it here.